orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -263,9 +276,8 @@ def backward(ctx, *grad_outputs):
         # Training model
         self._onnx_training = None
         self._training_session = None
-        self._training_io_binding = None
```

Comment:
What if we use a list of sessions to circumvent the need to push stuff into pytorch forward context that are not actually needed by the backward call? Using ctx seems a bit hacky

Reply:
What do you mean by list of sessions? Both training_iobinding and run_options are needed by backward call, in a way. run_options needs to kept in memory for backward call to run without error.

Reply:
Changed as we discussed offline